src/agoradatatools/__init__.py

```diff
@@ -0,0 +1,9 @@
+import sys
+import logging
+
+logging.basicConfig(
+    stream=sys.stdout,
+    level=logging.INFO,
+    format="INFO | %(asctime)s | %(message)s",
```

Comment:
Instead of INFO, can you have `%(levelname)s` also, can you add in `%(name)s`?

Reply:
Done. output looks like this now:
```
INFO: 2023-05-04 16:23:48 | agoradatatools.process | Agora Data Tools processing has started
INFO: 2023-05-04 16:23:48 | agoradatatools.process | Now processing genes_biodomains dataset
```

Reply:
Perfect, now if you do decide to add in logging in the transform modules, the module name will pop up.